cache/edge_lambdas/src/toggler.ts

```diff
@@ -8,10 +8,12 @@ import {
 type Test = {
   id: string;
   title: string;
-  //   range: [0, 50], // Run this test on 50% of users
-  //   range: [50, 100], // Run this test on 50% of users - not overlapping with ☝️
+  // Traffic included in the test is always split 50/50, the range specifies the percentage of total traffic to include in the test
+  // range: [0, 50], // Run this test on 50% of users
+  // range: [50, 100], // Run this test on 50% of users - not overlapping with ☝️
```

Comment:
is it "inclusive" 50 or "start after" 50 if it doesn't overlap?

Reply:
That's a good question I don't know the answer to, I'll have a look

Reply:
this is the relevent bit of code `const inRange = point >= from && point < to;` so up to but not including 50

Reply:
thanks!